docs/index.md

```diff
@@ -217,3 +217,14 @@ individuals, we have a number of open channels for communication.
 * To contribute code or documentation changes: please submit a [pull request on Github](https://github.com/docker/compose/pulls).
 
 For more information and resources, please visit the [Getting Help project page](https://docs.docker.com/project/get-help/).
+
+## Where to go next
+
+- [Installing Compose](install.md)
+- [Get started with Django](django.md)
+- [Get started with Rails](rails.md)
+- [Get started with WordPress](wordpress.md)
+- [Command line reference](./reference/index.md)
+- [Yaml file reference](yml.md)
+- [Compose environment variables](env.md)
+- [Compose command line completion](completion.md)
```

Comment:
I removed these because we already have this list above.  I think they got re-added in the rebase.

Could you remove them again?